goes-plume-viewer/src/components/plumeAnimation/index.jsx

```diff
@@ -43,7 +43,8 @@ export const PlumeAnimation = ({ plumes }) => {
                 handleAnimation(map, date, plumeDateIdxMap, plumes, bufferedLayer, bufferedSource);
             }
         });
-        map.addControl(timeline.current, "bottom-left");
+        const timelineElement = timeline.current.onAdd(map)
+        document.querySelector('#plume-animation-controller').append(timelineElement)
```

Comment:
As we have the #plume-animation-controller element defined in the jsx, we can use useRef to select the element in dom rather than using document querySelector